src/components/CommentsList.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from 'react'
+import { List, ListItem, Stack, Button, ButtonGroup } from '@mui/material';
+
+type CommentsListProps = {
+  type?: string,
+  list: any[],
+  actionHandler?: (c: string) => void,
+}
+
+export const CommentsList: FC<CommentsListProps> = ({
+  type,
+  list,
+  actionHandler,
+}) => {
+
+  const action = actionHandler ? actionHandler : () => {}
+
+  if(type === 'buttons') {
```

Comment:
it's not a big problem to have two diff components for such cases, since type buttons required some actions i thinks it's better separate them and create CommentsButtonsList and CommentsList with simple list